fcrepo-http-api/src/main/resources/views/common-node-actions.vsl

```diff
@@ -50,13 +50,23 @@
     <input type="text" id="new_id" placeholder="(auto-generated identifier)" name="slug"  class="form-control"/>
         </div>
 
+    <div class="form-group" id="turtle_payload_container">
+        <label for="rdf"  class="control-label">
+	        RDF (turtle)
+        </label>
+        <textarea rows="10" id="turtle_payload" name="turtle" class="form-control" 
+                  placeholder="@prefix dcmitype: <http://purl.org/dc/dcmitype/> 
+<> a dcmitype:Collection ."></textarea>
```

Comment:
Actually, the W3C states that newlines (CR & LF) are not permitted in placeholder attributes: https://www.w3.org/TR/html5/forms.html#the-placeholder-attribute